src/main/resources/assets/admin/common/js/dom/Element.ts

```diff
@@ -531,6 +531,12 @@ export class Element {
         return this.getEl().getAttribute('lang');
     }
 
+    setDir(value: string): Element {
+        this.getEl().setAttribute('dir', value); // doesn't work since reset.less in enonic-admin-artifacts...
+        this.getHTMLElement().style.direction = value; // ...sets direction style to ltr, so overwriting style
+        return this;
```

Comment:
Have released `enonic-admin-artifacts` 1.8.3 which no longer overrides `directory`. Let's upgrade to that (in both lib-admin-ui and CS) and remove the `style` override (and the comments).